pkgs/tools/package-management/nix/default.nix

```diff
@@ -226,11 +226,8 @@ in lib.makeExtensible (self: ({
   };
 
   nix_2_16 = common {
-    version = "2.16.2";
-    hash = "sha256-VXIYCDkvAWeMoU0W2ZI0TeOszCZA1o8trz6YCPFD5ac=";
-    patches = [
-      patch-rapidcheck-shared
-    ];
```

Comment:
This patch was already applied in 2.16.3

https://github.com/NixOS/nix/commit/45ac34e90caf98e33a02c0eeb0e6b19d02ef0677